index.js

```diff
@@ -12,32 +23,66 @@ const eth = new Eth(provider);
 
 var express = require('express'),
     cors = require('cors'),
-    app = express();
+    app = express(),
+    bodyParser = require('body-parser'),
+    jsonParser = bodyParser.json();
 
 app.use(cors());
 
-app.get('/address/:address', function(req, res, next){
-  eth.getTransactionCount(address, (err, nonce) => {
-    eth.sendTransaction({
-      from: address,
-      to: req.params.address,
-      gas: 100000,
-      value: (parseFloat(process.env.AMOUNT) || 1.5) * 1e18,
-      data: '0xde5f72fd', // sha3('faucet()')
-      nonce,
-    }, (err, txID) => {
-      if (err) {
-        console.log('Request failed', err)
-        return res.status(500).json(err)
+// Store issue ids and their bounty addresses
+var issueData = {};
+
+// Receive a POST request at the address specified by an env. var.
+app.post('/address/:address', jsonParser, function(req, res, next){
+  if (!req.body)
+    return res.sendStatus(400);
+  var commentBody = req.body.comment.body;
+  var issueId = req.body.issue.id;
+  var namePosition = commentBody.search("@" + name);
+  // Store toAddress from commiteth
+  if (namePosition == -1) {
+    issueData[issueId] = {"toAddress": commentBody.substring(commentBody.search("Contract address:") + 18, commentBody.search("Contract address:") + 60)}
```

Comment:
We should assert that it is the commiteth username who made the comment with the address